style.css

```diff
@@ -0,0 +1,36 @@
+
+.h1{
+  text-align:center;
+}
+
+header {
+  text-align: center;
+  background: white;
+  background-size: cover;
+  color: black;
+}
+
+
+body {
+  text-align: center;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+}
+
+.square {
+  width:100px;
+  height:100px;
+  color: red;
+  font-size: 20px;
+  text-align: center;
+  border:  5px solid red;
+
+}
+footer {
+  padding:0 0 2em 0;
+  display: inline-block;
+  text-align: center;
+  color: black;
+
```

Comment:
Nice CSS-ing! Just make sure you're actually using CSS — no stylistic HTML tags!